src/main/kotlin/com/terraformation/backend/customer/db/OrganizationStore.kt

```diff
@@ -175,6 +211,16 @@ class OrganizationStore(
         ?: throw UserNotFoundException(userId)
   }
 
+  fun fetchOrganizationIds(userId: UserId): List<OrganizationId> {
+    return dslContext
+        .select(ORGANIZATION_USERS.ORGANIZATION_ID)
+        .from(ORGANIZATION_USERS)
+        .where(ORGANIZATION_USERS.USER_ID.eq(userId))
+        .fetch(ORGANIZATION_USERS.ORGANIZATION_ID)
+        .filterNotNull()
+        .filter { currentUser().canListOrganizationUsers(it) }
```

Comment:
would this prevent contributors from deleting their org memberships?

Reply:
Good point. Fixed.